### ISSUE TITLE
Number of training steps

### ISSUE THREAD

**Issue by mhngu23** [kzl/decision-transformer#78]
Hi,

Thank you for the paper and your code.

I am writing to query about your decision on the number of training steps. Specifically, what is the reason behind choosing 10*10000 steps instead of a higher value, as I believe this will improve the performance in term of accuracy? 

Thank you for your attention!

**Comment by beapeak** [comment]
哥们，你跑通了吗
